build/server-error.js

```diff
@@ -12,12 +12,21 @@ const React = require('react');
 const RedBox = require('redbox-react').RedBoxError;
 const ReactDOMServer = require('react-dom/server');
 
-function renderError(error /*: any */) {
+function renderError(error /*: any = {} */) {
```

Comment:
What's the purpose for the `= {}` assignment here? If it's for a default assignment it seems like that should be outside of the comment?

Reply:
yes, default value.In this case the very last fallback for Object version of param will not faill if somebody tries to use undefined instead.
but outside of comment flow type check fails